src/sections/Home/CloudNativeManagement/index.js

```diff
@@ -63,7 +63,7 @@ const BannerDefault = () => {
               height="100%"
               style={{ margin: "auto" }}
               className="vidHero"
-            />
+            ><></></ReactPlayer>
```

Comment:
I added this because, it was giving an SSR (server-side rendering) issue, as this component requires a child, without that it was violating the SSR gatsby rule.